media/website/content/_index.md

```diff
@@ -592,58 +592,60 @@ The following fields need to be present in the config file:
 
 Example config:
 
-```
-[logging]
-log_level = INFO
-
-[paths]
-time_json_path = /etc/auditor_apel_plugin/time.json
-
-[intervals]
-report_interval = 86400
-
-[site]
-publish_since = 2023-01-01 13:37:42+00:00
-sites_to_report = {"SITE_A": ["site_id_1", "site_id_2"], "SITE_B": ["site_id_3"]}
-default_submit_host = gsiftp://accounting.grid_site.de:1337/jobs
-infrastructure_type = grid
-benchmark_type = hepscore23
-
-[auditor]
-auditor_ip = 127.0.0.1
-auditor_port = 3333
-auditor_timeout = 60
-benchmark_name = hepscore23
-cores_name = Cores
-cpu_time_name = TotalCPU
-cpu_time_unit = milliseconds
-nnodes_name = NNodes
-meta_key_site = site_id
-meta_key_submithost = headnode
-meta_key_voms = voms
-meta_key_username = subject
-
-[authentication]
-auth_url = https://msg.argo.grnet.gr:8443/v1/service-types/ams/hosts/msg.argo.grnet.gr:authx509
-ams_url = https://msg-devel.argo.grnet.gr:443/v1/projects/accounting/topics/gLite-APEL:publish?key=
-client_cert = /etc/grid-security/hostcert.pem
-client_key = /etc/grid-security/hostkey.pem
-ca_path = /etc/grid-security/certificates
-verify_ca = True
+```yaml
+logging:
+  log_level: INFO
+
+paths:
+  time_json_path: /etc/auditor_apel_plugin/time.json
+
+intervals:
+  report_interval: 86400
```

Comment:
Do you intend to add more options to the `logging`, `paths`, and `intervals` sections? If not, this could be simplified to just

```yaml
log_level: INFO
time_json_path: /etc/auditor_apel_plugin/time.json
report_interval: 86400
```

Reply:
Done